archinstall/lib/models/mirrors.py

```diff
@@ -0,0 +1,91 @@
+import datetime
+import pydantic
+import urllib.parse
+import urllib.request
+from typing import (
+	Dict,
+	List
+)
+
+from ..networking import ping, DownloadTimer
+from ..output import info, debug
+
+
+class MirrorStatusEntryV3(pydantic.BaseModel):
+	url :str
+	protocol :str
+	active :bool
+	country :str
+	country_code :str
+	isos :bool
+	ipv4 :bool
+	ipv6 :bool
+	details :str
+	delay :int|None = None
+	last_sync :datetime.datetime|None = None
+	duration_avg :float|None = None
+	duration_stddev :float|None = None
+	completion_pct :float|None = None
+	score :int|None = None
+	_latency :float|None = None
+	_speed :float|None = None
+	_hostname :str|None = None
+	_port :int|None = None
+
+	@property
+	def speed(self):
+		if self._speed is None:
+			info(f"Checking download speed of {self._hostname}[{self.score}] by fetching: {self.url}core/os/x86_64/core.db")
```

Comment:
`info` won't be shown to the user, so maybe just use `debug`

Reply:
I was thinking if the speed checks are slow, it's nice that they get printed during installation with `info()`:
![screenshot](https://github.com/user-attachments/assets/d2775957-83d7-4858-b3e4-6327673a3485)

It's shown between selecting the region - and returning to the menu, but just briefly.
But it does disrupt the look and feel of the menu system for sure, so we want to skip it?